conf/templates/common/additional-html-head.hbs

```diff
@@ -1 +1,3 @@
-<link rel="stylesheet" href="/assets/stylesheets/sunrise.css"/>
\ No newline at end of file
+<!-- start common/additional-html-head.hbs -->
+
+<!-- end common/additional-html-head.hbs -->
```

Comment:
Why did you change these files? (Same for `additional-html-scripts.hbs`)

Reply:
this was done by mistake 😓 
it will be reversed